tests/common.go

```diff
@@ -329,6 +329,57 @@ func ValidateAndDestroy(contexts []*scheduler.Context, opts map[string]bool) {
 	})
 }
 
+// CollectSupport creates a support bundle
+func CollectSupport() {
+	context(fmt.Sprintf("generating support bundle..."), func() {
+		Step(fmt.Sprintf("save all useful logs on each node"), func() {
+			nodes := node.GetWorkerNodes()
+			expect(nodes).NotTo(beEmpty())
+
+			for _, n := range nodes {
+				if !n.IsStorageDriverInstalled {
+					continue
+				}
+
+				// this task may hang so we wrapped in a timeout cmd
+				dumpTasksCmd := "timeout 10s bash -c \"echo t > /proc/sysrq-trigger\""
```

Comment:
There is a possibly that it might hang on cloud instances. Lets make it optional or take it out for now.